tsl/src/bgw_policy/drop_chunks_api.c

```diff
@@ -91,11 +223,70 @@ drop_chunks_add_policy(PG_FUNCTION_ARGS)
 				 errmsg("could not add drop_chunks policy because \"%s\" is not a hypertable",
 						get_rel_name(ht_oid))));
 
+	/* validate that the open dimension uses a time type */
+	open_dim = hyperspace_get_open_dimension(hypertable->space, 0);
+	partitioning_type = ts_dimension_get_partition_type(open_dim);
+
+	switch (older_than_type)
+	{
+		case INTERVALOID:
+			if (IS_INTEGER_TYPE(partitioning_type))
+				ereport(ERROR,
+						(errcode(ERRCODE_INVALID_PARAMETER_VALUE),
+						 errmsg(
+							 "interval older_than argument can only be used with hypertables "
+							 "that have non-integer time dimensions"))); // <<-- time columns? open
```

Comment:
time dimensions? time columns? open dimensions?

Reply:
this language is fine as is